test/unit/mdc-textfield/foundation.test.js

```diff
@@ -411,22 +395,22 @@ test('#updateOutline does nothing if no outline is present', () => {
 });
 
 test('on input styles label if input event occurs without any other events', () => {
-  const {foundation, mockAdapter, label} = setupTest();
+  const {foundation, mockAdapter} = setupTest();
   let input;
 
   td.when(mockAdapter.registerInputInteractionHandler('input', td.matchers.isA(Function)))
     .thenDo((evtType, handler) => {
       input = handler;
     });
   foundation.init();
-  td.verify(label.styleFloat(), {times: 0});
+  td.verify(mockAdapter.floatLabel(), {times: 0});
   input();
-  td.verify(label.styleShake(/* isValid */ true, /* isFocused */ true));
-  td.verify(label.styleFloat(/* value */ '', /* isFocused */ true, /* isBadInput */ false));
+  td.verify(mockAdapter.shakeLabel(/* isValid */ true, /* isFocused */ true));
```

Comment:
Let's check with the rest of the team on this, but I would much prefer to use a destructured object instead of individual boolean params.

E.g.:

```js
shakeLabel({isValid: true, isFocused: true})
```

Reply:
Seems like the group has spoken and prefers inline comments, what say you?